drivers/kscan/Kconfig.xpt2046

```diff
@@ -0,0 +1,11 @@
+# Copyright (c) 2023 Seppo Takalo
+# SPDX-License-Identifier: Apache-2.0
+
+config KSCAN_XPT2046
+	bool "XPT2046 resistive touch panel driver"
+	default y
+	depends on DT_HAS_XPTEK_XPT2046_ENABLED
+	select SPI
+	help
+	  Enable driver for Xptek XPT2046 resistive touch panel.
+	  This driver is very similar to ADS7843.
```

Comment:
There seem to be difference in channel numbering and XPT2046 seem to be able to report the touch pressure.
Otherwise, this same driver could be used, but I have no devices to try this, so I did not try to add support for both.